### DISCUSSION TITLE
Selector Based On All Atoms In Atom Family

### DISCUSSION THREAD

**Discussion by fjur** [facebookexperimental/Recoil#1766] [Q&A]
I want a selector to be based on all the values of an atom family. My use case is that I have a list of objects, and whenever a single object changes, I want to make the selector's value be false until I click a button or a delay happens.

How can I make it so the selector's value gets set to false whenever any atom in an atom family changes its value?

**Comment by drarmstr** [comment]
You may want to just wrap your atom family setter with a hook or function that also sets another atom with the "dirty" status that you can clear with the button or timeout.
